contrib/sly-package-fu.el

```diff
@@ -144,9 +155,12 @@ places the cursor at the start of the DEFPACKAGE form."
                                  (1+ (point))
                                (point))))))))))))
 
-(defun sly-export-symbols ()
+(defun sly-read-symbols ()
   "Return a list of symbols inside :export clause of a defpackage."
   ;; Assumes we're at the beginning of :export
```

Comment:
Is this assumption still valid?

Reply:
No, this function is generic and can be used inside of any s-exp. So, I've removed the comment.